gravitee/assets/logs/gravitee.yaml

```
@@ -0,0 +1,23 @@
+id: gravitee
+# See app_id in your integration's manifest.json file to learn more:
+# https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
+metric_id: gravitee
+# If for some reason id must be different than app_id, add the app_id in this field instead.
+# If id and app_id already match, this field can be left blank.
+backend_only: false
```

```suggestion
id: gravitee
metric_id: gravitee
backend_only: false
```
These comments are helpful during development and can be removed.

---

Removed those comments!